packages/app-form-builder/src/admin/plugins/editor/triggers/emailThanks.tsx

```diff
@@ -37,7 +37,11 @@ const plugin: FbEditorTrigger = {
                     </Cell>
                     <Cell span={12}>
                         <Bind name={"content"}>
-                            <Input rows={5} label={t`E-mail content`} />
+                            <Input
+                                rows={5}
+                                label={t`E-mail content`}
+                                description="You can use {fields.slug} (example {field.firstName}) in the email content, which will be replaced with actual values from the form before the email is sent."
```

Comment:
The description lacks consistency. First says `{fields.slug}` then provides `{field.firstName}` as an example (notice the `fields` / `field` difference).

Let's fix this and simplify:

> You can use {fields.FIELD_NAME} placeholders to reference form submission values. For example: "Hello {fields.firstName} {fields.lastName}!".